test/client/fixtures/runner/automation/select-element-test.js

```diff
@@ -412,6 +412,43 @@ $(document).ready(function () {
         }, 1500);
     });
 
+    // NOTE: Android and iOS ignores size and multiple attributes, all select elements behave like select with size=1
+    if (browserUtils.isSafari && browserUtils.hasTouchEvents || browserUtils.isAndroid) {
```

Comment:
let's move it to the var, something like `ignoreMultisizeSelectTests`. And we can write the comment when the var is declared once